nautobot_secrets_providers/providers/hashicorp.py

```diff
@@ -130,16 +140,19 @@ def get_client(cls, secret=None):
         # so we use a parameter to specify the path to the ca_cert, if not provided we use the default of None
         ca_cert = vault_settings.get("ca_cert", None)
 
-        namespace = vault_settings.get("namespace", None)
-
         # Get the client and attempt to retrieve the secret.
         try:
             if auth_method == "token":
                 client = hvac.Client(
-                    url=vault_settings["url"], token=vault_settings["token"], verify=ca_cert, namespace=namespace
+                    url=vault_settings["url"],
+                    token=vault_settings["token"],
+                    verify=ca_cert,
+                    namespace=vault_settings.get("namespace", None),
                 )
             else:
-                client = hvac.Client(url=vault_settings["url"], verify=ca_cert, namespace=namespace)
+                client = hvac.Client(
+                    url=vault_settings["url"], verify=ca_cert, namespace=vault_settings.get("namespace", None)
+                )
                 if auth_method == "approle":
```

Comment:
What is this change doing?

Reply:
Reducing the local variables. I needed a new variable `vault_config` which raised the number of local variables above the pylint limit.  I didn't see any reason for `namespace` to be a variable, so I just collapsed it into the function call.

Reply:
We're enforcing that too-many-locals rule? Ugh.

Reply:
I too would prefer to keep the old code and change the pylint config.